build.sbt

```diff
@@ -76,3 +128,19 @@ val root = project
   .in(file("."))
   .aggregate(backend, frontend)
   .aggregate(shared.projectRefs: _*)
+  .settings(
+    publish / skip := true,
+    addCommandAlias(
+      "herokuPush",
+      List(
+        "set backend/isHeroku := true",
+        "backend/Docker/publish",
+        "set backend/isHeroku := false",
```

Comment:
**issue:** looks like sbt generates some classes here that may get messed up on case-insensitive systems, will have to investigate this